runtime/queries/fga/highlights.scm

```diff
@@ -0,0 +1,50 @@
+(call_expression
+  function: (identifier) @function)
+
+(call_expression
+  function: (selector_expression
+    field: (identifier) @function.method))
+
+((type_identifier) @type.builtin
+  (#match? @type.builtin "^(string|int|map|uint|list|timestamp|bool|duration|double|ipaddress)$"))
+
+(condition_declaration
+  name: (identifier) @function)
+
+(version) @number
+
+[
+  "*"
+  "/"
+  "%"
+  ">>"
+  "<<"
+  "&"
+  "&^"
+] @operator
+
+[
+  "+"
+  "-"
+  "|"
+  "^"
+] @operator
+
+[
+  "or"
+  "and"
+  "but not"
+] @operator
```

Comment:
```suggestion
] @keyword.operator
```

Reply:
In general, you should be as specific as possible with these keywords.

Reply:
Tried to adjust the highlights as best as possible using the contributing guidelines. Thanks!